documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p><code>For a beginner's guide to TensorFlow Decision Forests, &nbsp;please refer to this tutorial.</code></p><p> </p><p>Is this tutorial a beginner's guide? If so, there's no need to link somewhere else. If not, then we can remove a bunch of introductory concepts from this guide and focus it more specifically. (Alternatively, reword this link such that it explains the the difference - e.g. "For an introduction to [TFDF] without Kaggle, please refer to this tutorial"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>